src/spec/validitygenerator.py

```diff
@@ -348,9 +348,9 @@ def createValidationLineForParameterIntroChunk(self, blockname, param, params, t
             if (lengths[0]) == 'null-terminated':
                 asciidoc += 'a null-terminated '
             elif (lengths[0]) == '1':
-                asciidoc += 'a pointer to '
+                asciidoc += 'a valid pointer to '
```

Comment:
This actually was how it was previously written, but the options are limited to:
"it's a pointer to X", "it's NULL", or "it's a nonsensical value"

The first is already implicitly "valid" as it implicitly is not the other two things.
The second is covered by stating that it "must: be NULL or...".
The third is covered by having no VU.

I'm not against the change particularly, in fact I coded it this way when I first added the implicit VU scripts, but I don't really see that it adds anything, and other group members have stronger opinions about this AFAIK.
What's your reasoning for this change?

Reply:
Well tell me how it goes with the board. There are way worse things than this being dropped.